app/models/user.rb

```diff
@@ -40,9 +40,9 @@ def suggested_videos
     rating = mood_rating
     if rating < 1.5
       VideoResults.new.get_suggested_videos('sad')
-    elsif rating > 1.5 && rating < 2.25
+    elsif rating >= 1.5 && rating < 2.25
```

Comment:
Good catch on this - glad I had a rating of 1.5 during our demo to catch this edge case